### ISSUE TITLE
Code Insights: Voice-over doesn't announce that insight was deleted 

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#37886]
#### Steps to reproduce:

1. Go to the dashboard page and turn on voice over 
2. Navigate to the delete insight modal 
3. Click the delete forever button
4. See (hear) no announcement about just deleted insight

#### Expected behavior:

When we delete insight we should hear announcements about it and also move focus to the dashboard (or next insight card element in the dashboard)

**Comment by sourcegraph-bot-2** [comment]
Heads up @joelkw @felixfbecker @vovakulikov @unclejustin - the "team/code-insights" label was applied to this issue.
